### PR TITLE
Brazilian Portuguese translation to Pretendo Network

### PR DESCRIPTION
**Pull request by Jao154**

### CODE REVIEW

File: locales/PT_br.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
+{
+	"nav": {
+		"about": "Sobre",
+		"faq": "FAQ",
+		"credits": "Créditos",
+		"progress": "Progresso"
+	},
+	"hero": {
+		"subtitle": "Game servers",
```
Comment (JipFr) [inline locales/PT_br.json:9]: Is this right?￼

File: locales/PT_br.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+			"Para o 3DS, também estamos trabalhando em Mario Kart 7, com o desejo de continuar trabalhando em outros jogos quando possível."
+		]
+	},
+	"faq": {
+		"title": "Perguntas Frequentes",
+		"text": "Aqui estão algumas perguntas comuns que nos são feitas para obter informações fáceis.",
+		"QAs": [
+			{
+				"pergunta": "O que é o Pretendo?",
+				"resposta": "Pretendo is an open source Nintendo Network replacement that aims to build custom servers for the Wii U and 3DS family of consoles. Our goal is to preserve the online functionality of these consoles, to allow players to continue to play their favorite Wii U and 3DS games to their fullest capacity."
```
Comment (JipFr) [inline locales/PT_br.json:36]: Looks like you missed a translation here￼

File: locales/PT_br.json [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+				"pergunta": "Se eu for banido no Nintendo Network, eu vou continuar banido no Pretendo?",
+				"resposta": "Não teremos acesso à Banimentos da Nintendo Network e nem todos os usuários serão banidos do nosso serviço. No entanto, teremos regras a seguir ao usar o serviço e o não cumprimento dessas regras pode resultar em banimento."
+			},
+			{
+				"pergunta": "Pretendo suportará Wii/Switch?",
+				"resposta": "O Wii já possui servidores personalizados fornecidos por <a href=\"https://wiimmfi.de/\" target=\"_blank\">Wiimmfi</a>. No momento, não desejamos dar suporte ao Switch, pois ele é pago e completamente diferente da Nintendo Network.
+			},
+			{
+				"pergunta": "Eu preciso de desbloqueio para conectar?",
+				"resposta": "Sim, você precisará desbloquear seu dispositivo para se conectar; no entanto, no Wii U você só precisará acessar o Homebrew Launcher (ou seja, Haxchi, Coldboot Haxchi ou mesmo o exploit do navegador da web), com informações sobre como o 3DS se conectará em breve."
```
Comment (JipFr) [inline locales/PT_br.json:64]: There is some invalid JSON here￼

**Review by JipFr — CHANGES_REQUESTED** [review]

File: locales/PT_br.json [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+				"answer": "Pretendo é projetado principalmente para o hardware Wii U e 3DS; no momento, o único emulador para esses consoles com suporte para NN é o Cemu. O Cemu não oferece suporte oficialmente a servidores personalizados, mas ainda deve ser possível usar o Pretendo com o Cemu. <br> O Pretendo atualmente não oferece suporte ao Cemu. "
+			},
+			{
+				"question": "Se eu for banido no Nintendo Network, eu vou continuar banido no Pretendo?",
+				"answer": "Não teremos acesso à Banimentos da Nintendo Network e nem todos os usuários serão banidos do nosso serviço. No entanto, teremos regras a seguir ao usar o serviço e o não cumprimento dessas regras pode resultar em banimento."
+			},
+			{
+				"question": "Pretendo suportará Wii/Switch?",
+				"answer": "O Wii já possui servidores personalizados fornecidos por <a href=\"https://wiimmfi.de/\" target=\"_blank\">Wiimmfi</a>. No momento, não desejamos dar suporte ao Switch, pois ele é pago e completamente diferente da Nintendo Network.
+			},
```
Comment (mrjvs) [inline locales/PT_br.json:61]: There is an end quote missing on line 61

**Review by mrjvs — CHANGES_REQUESTED** [review]
can be merged after the json file is in proper syntax (check the review comments)

File: locales/PT_br.json [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+	},
+	"credits": {
+		"title": "O Time",
+		"text": "Conheça o time de desenvolvedores por trás do projeto",
+		"people": [
+			{
+				"name": "Jonathan Barrow (RedDucks)",
+				"caption": "Criador do projeto e desenvolvedor chefe",
+				"picture": "https://github.com/redduckss.png",
+				"github": "https://github.com/RedDuckss"
```
Comment (mrjvs) [inline locales/PT_br.json:76]: change to this. he has a new username
```suggestion
				"name": "Jonathan Barrow (jonbarrow)",
				"caption": "Criador do projeto e desenvolvedor chefe",
				"picture": "https://github.com/jonbarrow.png",
				"github": "https://github.com/jonbarrow"
```

**Comment by montylion** [comment]
Closed in favor of #57. You're welcome to provide feedback to the accepted localization.

**Pull request closed without merging**